docs/docs/guides/customizing-the-stream-extractor.md

```diff
@@ -20,6 +29,40 @@ class FileConnector(StreamConnector, alias="file"):
 
 The `alias` class attribute is used to specify the name of the stream connector. This name is used in the `stream` configuration option of the `StreamExtractor` to specify which stream connector to use. For more information see the [Stream Extractor](../reference/extractors.md) reference.
 
+### Registering the Stream Connector
+
+Stream connectors are registered via the [entry_points](https://setuptools.pypa.io/en/latest/userguide/entry_point.html#entry-points-for-plugins) API of a Python Package. Specifically, the `entry_point` named `stream_connectors` inside of the `nodestream.plugins` group is loaded. It is expected to be a subclass of `nodestream.pipeline.extractors.streams:StreamConnector` as directed above.
```

Comment:
Here you switched to "It is expected" rather than "Value Provider" which makes me think the previous docs contained some copy pasta.

Reply:
Indeed.